<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Movie</class>
 <widget class="QMainWindow" name="Movie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add new movies into DB</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>491</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color:rgb(43, 89, 99);
font: 900 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <column>
     <property name="text">
      <string>Title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Desc</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Genre</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>491</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="titlelineEdit">
         <property name="styleSheet">
          <string notr="true">background-color: white;
color:rgb(43, 89, 99);
font: 900 9pt &quot;Segoe UI&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Genre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="GenrelineEdit_2">
         <property name="styleSheet">
          <string notr="true">background-color: white;
color:rgb(43, 89, 99);
font: 900 9pt &quot;Segoe UI&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Desc</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="desclineEdit_3">
         <property name="styleSheet">
          <string notr="true">background-color: white;
color:rgb(43, 89, 99);
font: 900 9pt &quot;Segoe UI&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>141</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="deletebtn">
       <property name="styleSheet">
        <string notr="true">background-color: white;
color:rgb(43, 89, 99);
font: 900 9pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Delete data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="insertbtn">
       <property name="styleSheet">
        <string notr="true">background-color: white;
color:rgb(43, 89, 99);
font: 900 9pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Insert Into Db</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updatebtn">
       <property name="styleSheet">
        <string notr="true">background-color: white;
color:rgb(43, 89, 99);
font: 900 9pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Update Db</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="displaybtn">
       <property name="styleSheet">
        <string notr="true">background-color: white;
color:rgb(43, 89, 99);
font: 900 9pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Display Db</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
